features/admin.roles.v2/components/edit-role/edit-role-users.tsx

```diff
@@ -132,10 +132,17 @@ export const RoleUsersList: FunctionComponent<RoleUsersPropsInterface> = (
     };
 
     useEffect(() => {
-        const defaultSelectedUserStore: string = activeUserStore ??
-            disabledUserstores.includes(RemoteUserStoreConstants.PRIMARY_USER_STORE_NAME) ? "DEFAULT" : "PRIMARY";
+        if (activeUserStore) {
+            setSelectedUserStoreDomainName(activeUserStore);
+        } else {
+            const defaultSelectedUserStore: string = disabledUserstores.includes(
+                RemoteUserStoreConstants.PRIMARY_USER_STORE_NAME
+            )
+                ? "DEFAULT"
```

Comment:
Are there constants for these?

Reply:
Fixed with [49562dc](https://github.com/wso2/identity-apps/pull/7015/commits/49562dc893f0d99ffd82a98cf055e50b64381f53)